docs/ml/upload-model.md

```diff
@@ -39,34 +39,44 @@ See [ML models in the registry](/registry/#ml-models) for more information.
 Select **Private** to only publish the model for internal use within this {{< glossary_tooltip term_id="organization" text="organization" >}}.
 Select **Next steps** to continue.
 
-{{<imgproc src="/ml/upload-model.png" resize="600x" alt="Upload model menu on the DATA tab of the Viam app.">}}
+{{<imgproc src="/ml/upload-model.png" resize="900x" alt="Upload model menu on the DATA tab of the Viam app." style="max-width: 700px">}}
```

Comment:
Something strange is happening where the image spills out of the tab box 
![image](https://github.com/viamrobotics/docs/assets/75634662/abcb668f-11e5-40cd-ab55-bab8fffece18)
Not a big deal in this case but maybe try 
```suggestion
{{<imgproc src="/ml/upload-model.png" resize="900x" alt="Upload model menu on the DATA tab of the Viam app." style="max-width: 600px">}}
```
instead?